src/main/java/teammates/storage/sqlentity/questions/FeedbackTextQuestion.java

```diff
@@ -0,0 +1,50 @@
+package teammates.storage.sqlentity.questions;
+
+import teammates.common.datatransfer.questions.FeedbackTextQuestionDetails;
+import teammates.storage.sqlentity.FeedbackQuestion;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Converter;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Transient;
+
+/**
+ * Represents a FeedbackTextQuestion entity.
+ */
+@Entity
+public class FeedbackTextQuestion extends FeedbackQuestion {
+
+    @Column(nullable = false)
+    @Convert(converter = FeedbackTextQuestionDetailsConverter.class)
+    private String questionDetails;
```

Comment:
If the convert annotation works as expected, you shouldn't have to use string here.

e.g. should be private XXQuestionDetails questionDetails;

The converter will convert this to string in the db.

Reply:
ah right, i overlooked that, changed!